pandas/tests/io/excel/test_readers.py

```diff
@@ -1278,6 +1278,24 @@ def test_ignore_chartsheets_by_int(self, request, read_ext):
         ):
             pd.read_excel("chartsheet" + read_ext, sheet_name=1)
 
+    def test_dtype_dict_unchanged_with_duplicate_columns(self, read_ext):
+        # GH 42462
+
+        filename = "test_common_headers" + read_ext
```

Comment:
can you not use existing data? or simply do a round trip; i don't want to add even more files like this